library/src/network.rs

```diff
@@ -13,7 +13,7 @@ use crate::config::{current_arch, current_platform, UpdateConfig};
 
 // https://stackoverflow.com/questions/67087597/is-it-possible-to-use-rusts-log-info-for-tests
 #[cfg(test)]
-use std::println as debug; // Workaround to use println! for logs.
+use std::{println as info, println as debug}; // Workaround to use println! for logs.
```

Comment:
Is this line necessary since we're changing the "sending patch check request" log to an `info!`?

Reply:
Correct.  Previously info! wasn't used in this file.

Reply:
Ah! LGTM then.